.github/workflows/ci_tests.yaml

```diff
@@ -79,6 +79,14 @@ jobs:
             pandas-version: ''
             xarray-version: ''
             optional-packages: ' contextily geopandas ipython pyarrow rioxarray sphinx-gallery'
+          # The job below is for testing geopandas v0.x on Ubuntu. Python 3.11 is used
```

Comment:
> This PR adds a separate CI job in which geopandas v0.x is installed. The CI job has to use Python 3.11, otherwise other Python 3.10/3.12 jobs will be override by this one. The job also only runs on Ubuntu, to avoid using too many CI resources.

Could we test geopandas v0.x in [`ci_tests_legacy.yaml`](https://github.com/GenericMappingTools/pygmt/blob/main/.github/workflows/ci_tests_legacy.yaml) instead? Or do we want to test it more regularly than that?

Reply:
We can, but in coverage reports the geopandas v0.x would be uncovered since the Legacy workflow doesn't produce coverage reports.